src/main/java/uk/gov/justice/digital/hmpps/cmd/api/service/DryRunNotificationService.kt

```diff
@@ -140,7 +113,7 @@ class DryRunNotificationService(
     item: CsrModifiedDetailDto,
     first: Boolean
   ): CsrModifiedDetailDto =
-    if (first) item else if (item.shiftModified!!.isAfter(acc!!.shiftModified)) item else acc
+    if (first || item.shiftModified == null || acc == null || item.shiftModified.isAfter(acc.shiftModified)) item else acc
```

Comment:
Have improved this. acc has to be nullable as this is in the parameter profile of the .aggregate fun